server/commands/connected_objects.go

```diff
@@ -92,10 +161,16 @@ func (c *ConnectedObjectsCommand) streamedConnectedObjects(
 					StoreID:    storeID,
 					ObjectType: req.ObjectType,
 					Relation:   req.Relation,
-					User: &openfgapb.ObjectRelation{
-						Object:   req.User.Object,
-						Relation: innerLoopIngress.Ingress.GetRelation(),
+					User: &UserRefObjectRelation{
+						ObjectRelation: &openfgapb.ObjectRelation{
+							Object:   targetUserObj,
+							Relation: innerLoopIngress.Ingress.GetRelation(),
+						},
 					},
+					// User: &openfgapb.ObjectRelation{
```

Comment:
Do we intend to have this commented out code?

Reply:
Nope, that was residual. Thanks for pointing it out 👍  I've removed it